src/tracker-component.js

```diff
@@ -66,15 +66,32 @@ type Config = {|
     paramsToBeaconUrl? : ParamsToBeaconUrl
 |};
 
+const getUserId = () => {
+    return document.cookie ? document.cookie : localStorage.getItem('user-id');
```

Comment:
should we make the localStorage key something that is less likely to be set by someone else?

Reply:
Also why are we returning all cookies?

Reply:
Please null-check `localStorage`. It's not available all the time -- for some browsers, in private mode, it will not be defined.

Reply:
I check for if the value returned is truthy here and if it's not truthy then I set localStorage so that it's always a value https://github.com/paypal/paypal-muse-components/pull/4/files#diff-bc1bab447af0f7377f922001d7fb757dR91
let me know if that's what you mean

Reply:
https://jonathanmh.com/use-ios-safari-localstorage-sessionstorage-private-mode/

Reply:
thanks for the information, we change the implementation to only use cookies